applications/app/views/signup/newsletterContent.scala.html

```diff
@@ -36,9 +36,12 @@
 
 
                 <div class="signup-confirmation is-hidden js-signup-confirmation">
-                    @fragments.inlineSvg("envelope", "icon")
-                    <div class="signup-confirmation__message">
-                        <h3 class="signup-confirmation--success">Check your Inbox!</h3>
+                    <div class="signup-confirmation__success-headline-container">
+                        @fragments.inlineSvg("envelope", "icon")
+                        <span class="signup-confirmation--success">@Html(emailListing.emailEmbed.successHeadline)</span>
```

Comment:
Is there a way to keep the same styling and use of tags as before? I haven't tested this but my guess is the `h3` will be better for accessibility.

Also, it seems we've changed where the background grey sits which has the effect of making the confirmation message look a bit buttony, which might entice people to try and click on it.

## Before
<img width="171" alt="Screenshot 2022-02-28 at 15 02 09" src="https://user-images.githubusercontent.com/1336821/156005880-e71006ab-ee58-49ba-a7b5-0d4c9729d6f2.png">

## After
<img width="118" alt="Screenshot 2022-02-28 at 15 02 29" src="https://user-images.githubusercontent.com/1336821/156005875-b92e86d8-1777-4de8-8ac8-4db57299a8e0.png">

Reply:
Hi Oliver - thanks, that's a fair point. The mark-up and styling does need to be somewhat different from before as there are two lines of confirmation text now, but I'm sure I can change the appearance of the first to be less button-y and use a h3.

Reply:
That's done - I've added new screenshot